website/_enemies/generic/key.md

```diff
@@ -1,12 +1,12 @@
 ---
-name: Key (for Door)
+name: Key
 enemygroup: generic
 can_hurt: false
 can_path: true
 hp: 1
 sprite_folder_name: key
 ---
 
-When killed, will fly to its associated [Door](#enemy-door-for-key) and kill it.
+When killed, will fly to its associated [Door](#enemy-door) and unlock it, making it non-solid so that entities may pass through.
```

Comment:
> making it non-solid so that entities may pass through

What does this mean?

Reply:
I was trying to say that the door goes away, but in more official sounding terms. maybe this would be better?

```suggestion
When killed, will fly to its associated [Door](#enemy-door) and unlock it, causing the door to open so that entities may pass through.
```

Reply:
That sounds much better.